pages/linux/chpasswd.md

```diff
@@ -0,0 +1,20 @@
+# chpasswd
+
+> Change passwords for multiple users via taking input from stdin.
+> More information: <https://manpages.debian.org/jessie/passwd/chpasswd.8.en.html>.
+
+- Change password for a user:
+
+`printf "{{username}}:{{new_password}}" | sudo chpasswd`
+
+- Change passwords for multiple users (Mind the lack of space in the stdin text):
+
+`printf "{{username_one}}:{{new_password_one}}\n{{username_two}}:{{new_password_two}}" | sudo chpasswd`
+
+- Change password into the supplied encrypted password for a user:
+
+`printf "{{username}}:{{new_encrypted_password}}" | sudo chpasswd --encrypted`
+
+- Change password for a user and use a specific cryptographic method to encrypt the password to be stored:
+
+`printf "{{username}}:{{new_password}}" | sudo chpasswd --crypt-method {{NONE|DES|MD5|SHA256|SHA512}}`
```

Comment:
```suggestion
- Change the password for a specific user, and use a specific encryption for the stored password (defaults to `DES`):

`printf "{{username}}:{{new_password}}" | sudo chpasswd --crypt-method {{DES|NONE|MD5|SHA256|SHA512}}`
```

Reply:
If I am understanding the docs correctly, I believe the default encryption method for `chpasswd` varies across Linux distros. Example: [Arch uses `DES` by default](https://manned.org/man/arch/chpasswd.8#:~:text=of%20these%20values%3A-,des%20(default),-%2C%20MD5%2C%20SHA256%2C%20SHA512) and [Debian uses `PAM`  by default](https://manned.org/man/debian-bullseye/chpasswd.8#:~:text=by%20default%2C%20pam%20is%20used%20to%20encrypt%20the%20passwords.).

Could we omit the details on the default method, and let the user look at the man page for more info on it?

Reply:
Good catch. Sure, let's leave it out.